src/exports.js

```diff
@@ -115,7 +129,7 @@ const makeExports = (useImportant, selectorHandlers) => {
         StyleSheetServer,
         StyleSheetTestUtils,
 
-        css(...styleDefinitions) {
+        css(...styleDefinitions /* : SheetDefinition[] */) {
```

Comment:
Is the conditional style API still supported?

https://github.com/Khan/aphrodite#conditionally-applying-styles

It might be worth widening the definition to accept at least the boolean `false` here, as was done in the flow-typed definition.

https://github.com/flowtype/flow-typed/commit/c47ccdab811f52e1afb68492689330421030b344